sdks/python/apache_beam/io/aws/s3io.py

```diff
@@ -157,8 +157,9 @@ def list_prefix(self, path, with_metadata=False):
       else:
         break
 
-    logging.info(
-        "Finished listing %s files in %s seconds.",
+    logging.log(
+        # do not spam logs when list_prefix is likely used to check empty folder
+        logging.INFO if counter > 0 else logging.DEBUG,
```

Comment:
did you intentionally omit the `"Finished listing %s files in %s seconds.",` here?

Reply:
my bad. Fixing it, will double check changes